bundler/lib/bundler/cli/gem.rb

```diff
@@ -238,15 +255,45 @@ def ask_and_set_test_framework
       test_framework
     end
 
-    def test_framework_hint
-      if Bundler.settings["gem.test"] == false
+    def hint_text(setting)
+      if Bundler.settings["gem.#{setting}"] == false
         "Your choice will only be applied to this gem."
       else
         "Future `bundle gem` calls will use your choice. " \
-        "This setting can be changed anytime with `bundle config gem.test`."
+        "This setting can be changed anytime with `bundle config gem.#{setting}`."
       end
     end
 
+    def ask_and_set_ci
+      ci_template = options[:ci] || Bundler.settings["gem.ci"]
+
+      if ci_template.to_s.empty?
+        Bundler.ui.confirm "Do you want to set up automated testing for your gem? " \
```

Comment:
I feel like we are saying to much, we don't need to tell them what is CI used for

Maybe just:

```
      "Do you want to configure a continuous integration service?\n"\

      "Supported services:"\
      "* CircleCI:       https://circleci.com/\n" \
      "* GitHub Actions: https://github.com/features/actions\n" \
      "* GitLab CI:      https://docs.gitlab.com/ee/ci/\n" \
       "* Travis CI:      https://travis-ci.org/\n" \
          "\n"
```